BasicSelenium.py

```diff
@@ -0,0 +1,59 @@
+from selenium import webdriver
+import unittest
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class Test(unittest.TestCase):
+
+    def basicSelenium(self):
+        browser = webdriver.Chrome('C:\webdriver\chromedriver.exe')
+        wait = WebDriverWait(browser, 10)
+
+        manCategory = (By.CSS_SELECTOR, ".sub-menu:nth-child(2)")
+        FirstItem = (By.CLASS_NAME, "ems-prd-inner")
+        sizeSelectButton = (By.CSS_SELECTOR, ".dropdownrow:nth-child(1)")
+        addToCartButton = (By.CLASS_NAME, "urunDetay_btnSepeteAt")
+        selectCart = (By.CLASS_NAME, "ems-clone-mini-quatity")
+        goToCart = (By.ID, "plhKSepet_sepetLink")
+        backToHomepageButton = (By.CLASS_NAME, "sepetLogo")
+
+        browser.get("https://www.spx.com.tr/")  # Go to website
+        browser.maximize_window()  # maximize window to prevent possible responsiveness issues
+
+        wait.until(EC.presence_of_element_located(manCategory)).click()  # go to category page
+
+        categoryAssertText = browser.find_element_by_class_name(

```

Comment:
burda assert için kullanacağın locatorları yukarıda tanımlayıp kullanman gerek. find_element değil de diğerleri gibi presence_of_element_located vs. şeklinde kullanabilirsin